planner/core/find_best_task.go

```diff
@@ -691,7 +691,7 @@ func (ds *DataSource) findBestTask(prop *property.PhysicalProperty, planCounter
 				p: dual,
 			}, cntPlan, nil
 		}
-		canConvertPointGet := (!ds.isPartition && len(path.Ranges) > 0) || (ds.isPartition && len(path.Ranges) == 1)
+		canConvertPointGet := len(path.Ranges) > 0
```

Comment:
@lysu For partition table, multi-range can also triger `Get`/`BatchGet` operation, right?

Reply:
I guess maybe some reason let https://github.com/pingcap/tidb/pull/14775#discussion_r388731250 only support single range? @lzmhhh123

Reply:
Sorry, I forgot the reason. Maybe the previous point get can't handle the partition table.

Reply:
oh...current point get can handle the partition